core/feconf.py

```diff
@@ -953,6 +953,10 @@ def get_empty_ratings() -> Dict[str, int]:
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_THREADLIST_URL_PREFIX_FOR_TOPICS = '/threadlisthandlerfortopic'
 FEEDBACK_THREAD_VIEW_EVENT_URL = '/feedbackhandler/thread_view_event'
+FEEDBACK_UPDATES_DATA_URL = (
+    '/feedbackupdateshandler/data')
+FEEDBACK_UPDATES_URL = '/feedback-updates'
```

Comment:
```suggestion
FEEDBACK_UPDATES_URL = '/feedbackupdates'
```

Reply:
The changes are not done? I do not see the changes here. Just a general note, Please do not resolve the conversation as it makes it hard for the reviewers to verify the changes and the comments they previously did.
Thanks

Reply:
Hi, @lkbhitesh07 , I committed the suggestion 10 hours ago. Please see here 
(https://github.com/oppia/oppia/pull/18350/commits/102c35371d2dc626159db98c54fafad6b28e65f3). 
Please let me know if I am wrong.
Sorry but I have not resolved the comment, when I have committed the suggestion, the comment gets resolved automatically.
Thanks!

Reply:
There might be some problem from my end, thanks for the changes @SubhashThenua, I have approved it.